### ISSUE TITLE
Refactor reply logic and use the reply information on other commands too

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#240]
```
The reply logic should be taken out of the sms command class into a new class, 
so that the information can be used on various other commands (e.g. the dial 
command).
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 30 Dec 2011 at 11:17
- Blocking: #260

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 31 Mar 2012 at 10:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 13 Apr 2012 at 7:26
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Done: Command class RecipientCmd handles the last recipient.
```

Original comment by `fschm...@gmail.com` on 11 Nov 2012 at 11:23
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 4.3 release
```

Original comment by `fschm...@gmail.com` on 26 Jan 2013 at 5:45
- Changed state: **Deployed**
